src/convertToObject.js

```diff
@@ -6,7 +6,20 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const styles = sourceString
+    .split(';')
+    .map((style) => style.split(':'))
+    .reduce((acc, [key, value]) => {
+      if ((key, value)) {
+        const [_key, _value] = [key.trim(), value.trim()];
+
+        acc[_key] = _value;
+      }
+
+      return acc;
```

Comment:
[CODE STYLE] - The reduce function's accumulator variable name `acc` could be more descriptive. Consider renaming it to something like `styleObj` or `cssObj` to clearly indicate its purpose.